### PR TITLE
add resp3 protocol part 1(add data type and implement some simple)

### PR DESCRIPTION
**Pull request by Chyroc**
# Pull Request for issue #

#76 

## Description

* [x] simple string
* [x] blob string
* [x] simple error
* [x] blob error
* [x] number
* [x] null
* [x] boolean
* [ ] verbatim string
* [ ] double
* [ ] big number
* [ ] array
* [ ] map
* [ ] set
* [ ] push
* [ ] attribute

### CODE REVIEW

File: internal/protcl/errors.go [hunk 1]
```diff
@@ -101,3 +101,21 @@ func (e *ErrUnknownCommand) Error() string {
 func (ErrUnknownCommand) Recoverable() bool {
 	return true
 }
+
+// ErrProtocolType is for protocol type error
+type ErrProtocolType struct {
+	Type byte
+}
+
```
Comment (kasvith) [inline internal/protcl/errors.go]: If these errors should not disconnect client, you can implement Recoverable interface on them

Comment (Chyroc) [reply internal/protcl/errors.go]: done

**Comment by kasvith** [comment]
Lint errors :)

**Comment by kasvith** [comment]
Tnx @Chyroc

**Pull request merged**